source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,31 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you are a reseller, have a non-standard contract, or want to adjust the number of active users on your license please contact sales@mattermost.com.
+
+System Admins will be alerted 60 days prior to the license expiry via a banner in Mattermost. Select **Renew license now** to start the renewal process in the Customer Portal. You can also dismiss the banner and renew your license at a later date via **System Console > Edition and License**.
+
+The renewal page in the Customer Portal lists your license information and contact details which are pre-populated based on your existing license subscription. 
+
+**To process your license renewal:**
+
+1. Enter your contact and billing information.
```

Comment:
Let's match these fields with the renewal form, e.g.

```
1. Enter your **Account Details**, **Additional Contact** and **Payment Details**.
```

Reply:
```suggestion
1. Enter your **Account Details**, **Additional Contact**, and **Payment Details**.
```